synapse/models/entity.py

```diff
@@ -157,21 +157,6 @@
                     ),
                 },
 
-                'aliases': (
```

Comment:
These alias deletes ( and the ones below ) are removing vestigial things i was prototyping.